junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.jupiter.params;
+
+import static org.junit.platform.commons.meta.API.Usage.Experimental;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.platform.commons.meta.API;
+
+@Target({ ElementType.ANNOTATION_TYPE, ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@API(Experimental)
+@TestTemplate
+@ExtendWith(ParameterizedTestExtension.class)
+public @interface ParameterizedTest {
+
+	String name() default "[{index}] {arguments}";
```

Comment:
I thought about using `@DisplayName`. But then the container descriptor for the method would also get that display name. IMHO that doesn't make sense so I decided to do it differently, in a way like we do for `TestFactory` methods.